### DISCUSSION TITLE
SEB Crashes unexpectedly very often

### DISCUSSION THREAD

**Discussion by leumas700** [SafeExamBrowser/seb-win-refactoring#924] [Q&A]
We are using moodle with seb at Berufsfachschule Baden since the start of this year and it mostly works pretty good. Since this weeks we have a lot of problems with seb, it crashed alot (around 10 out of 15 students during an exam). I atteched some logfiles of affected students. What i found what could be helpful so far is this:
- ERROR: Client application has unexpectedly terminated with exit code 0!
- ERROR: [ClientProxy] Failed to ping host!

I attached some logfiles. I would be very thankful for feedbacks asap since teacher already start to disable SEB on their exams because of this.
[logs.zip](https://github.com/user-attachments/files/15902680/logs.zip)

**Comment by dbuechel** [comment]
This is most definitely due to #883. Please update to version 3.7.1 as mentioned in the issue.

**Reply by leumas700** [reply]
Thank you for the fast response. We will update accordingly and hope this helps.
